lib/kql/kql/__init__.py

```diff
@@ -45,12 +45,12 @@ def to_eql(text, optimize=True, schema=None):
     return converted.optimize(recursive=True) if optimize else converted
 
 
-def parse(text, optimize=True, schema=None):
+def parse(text, optimize: bool = True, schema: dict = None, normalize_kql_keywords: bool = True):
```

Comment:
We should leave the default behavior as is

```suggestion
def parse(text, optimize: bool = True, schema: dict = None, normalize_kql_keywords: bool = False):
```